### PR TITLE
schema outline

### PR DESCRIPTION
**Pull request by clv1**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,6 +1,22 @@
```
Comment (IshikaM19) [inline schema.sql:1]: Maybe add 'USE Plants; go' here so that it makes these tables directly in the plants database

File: schema.sql [hunk 2]
```diff
@@ -10,13 +26,39 @@ CREATE TABLE s_alpha.recording_event (
...
+
+CREATE TABLE s_alpha.origin_location (
+    origin_location_id BIGINT IDENTITY(1,1) NOT NULL,
+    longitude FLOAT NOT NULL,
+    latitude FLOAT NOT NULL,
+    town VARCHAR(50) NOT NULL,
+    country VARCHAR(50) NOT NULL,
+    country_abbreviation VARCHAR(10) NOT NULL,
+    continent VARCHAR(50) NOT NULL,
+    PRIMARY KEY (origin_location_id)
```
Comment (BenBeard115) [inline schema.sql:54]: This needs to be made before the plant table and the plant table needs this to be added as a foreign key

File: schema.sql [hunk 3]
```diff
@@ -10,13 +26,39 @@ CREATE TABLE s_alpha.recording_event (
...
-go
+
+
+CREATE TABLE s_alpha.origin_location (
+    origin_location_id BIGINT IDENTITY(1,1) NOT NULL,
+    longitude FLOAT NOT NULL,
+    latitude FLOAT NOT NULL,
+    town VARCHAR(50) NOT NULL,
+    country VARCHAR(50) NOT NULL,
+    country_abbreviation VARCHAR(10) NOT NULL,
```
Comment (BenBeard115) [inline schema.sql:52]: could limit this more if you'd like?

File: schema.sql [hunk 4]
```diff
@@ -10,13 +26,39 @@ CREATE TABLE s_alpha.recording_event (
...
+
+
+-- go
 
 -- INSERT INTO s_alpha.recording_event
 --     (plant_id, botanist_id, soil_moisture, temperature, recording_taken, last_watered) 
 -- VALUES 
---     ();
-
--- go
```
Comment (BenBeard115) [inline schema.sql:22]: You can scrap all of this

File: schema.sql [hunk 5]
```diff
@@ -1,6 +1,22 @@
...
+GO
+
+CREATE TABLE s_alpha.plant (
+    plant_id INT IDENTITY(1,1) NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    scientific_name VARCHAR(100) NOT NULL,
+    origin_location_id INT NOT NULL,
+    image_url VARCHAR(300) NOT NULL,
+    PRIMARY KEY (plant_id)
+);
```
Comment (IshikaM19) [inline schema.sql:20]: I'm just wondering, do you have to write GO after each CREATE TABLE statement?

File: schema.sql [hunk 6]
```diff
@@ -1,6 +1,22 @@
...
+DROP TABLE s_alpha.origin_location;
+
+GO
+
+CREATE TABLE s_alpha.plant (
+    plant_id INT IDENTITY(1,1) NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    scientific_name VARCHAR(100) NOT NULL,
+    origin_location_id INT NOT NULL,
+    image_url VARCHAR(300) NOT NULL,
```
Comment (BenBeard115) [inline schema.sql:17]: Some plants don't have an image, so some will be NULL

File: schema.sql [hunk 7]
```diff
@@ -10,13 +26,39 @@ CREATE TABLE s_alpha.recording_event (
...
+    FOREIGN KEY (plant_id) REFERENCES plant(plant_id),
+    FOREIGN KEY (botanist_id) REFERENCES botanist(botanist_id)
+    );
+
+
+CREATE TABLE s_alpha.botanist (
+    botanist_id BIGINT IDENTITY(1,1) NOT NULL,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    email VARCHAR(100) NOT NULL,
```
Comment (IshikaM19) [inline schema.sql:39]: Love that you've made specific constraints on the length of emails and names. Maybe worth noting this down in the Readme and how you came to that conclusion :)

File: schema.sql [hunk 8]
```diff
@@ -1,6 +1,21 @@
...
+DROP TABLE s_alpha.plant;
+DROP TABLE s_alpha.recording_event;
+DROP TABLE s_alpha.botanist;
+DROP TABLE s_alpha.origin_location;
+GO
 
+CREATE TABLE s_alpha.plant (
+    plant_id INT IDENTITY(1,1) NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    scientific_name VARCHAR(100) NOT NULL,
```
Comment (BenBeard115) [inline schema.sql:13]: Some scientific names are also missing so they are allowed to be NULL too

File: schema.sql [hunk 9]
```diff
@@ -10,13 +26,39 @@ CREATE TABLE s_alpha.recording_event (
...
+    email VARCHAR(100) NOT NULL,
+    phone_number VARCHAR(30) NOT NULL,
+    PRIMARY KEY (botanist_id)
     );
 
-go
+
+
+CREATE TABLE s_alpha.origin_location (
+    origin_location_id BIGINT IDENTITY(1,1) NOT NULL,
```
Comment (clv1) [inline schema.sql:47]: This needs to be INT, or you'll have issues when repositioning.

File: schema.sql (hunk 2 above)
Comment (clv1) [reply schema.sql:54]: Great catch!

File: schema.sql [hunk 10]
```diff
@@ -1,23 +1,58 @@
...
-go
+GO
+
+DROP TABLE s_alpha.plant;
 DROP TABLE s_alpha.recording_event;
+DROP TABLE s_alpha.botanist;
+DROP TABLE s_alpha.origin_location;
+GO
+
+
```
Comment (anukrr) [inline schema.sql:10]: nice organisation.

**Review by anukrr — DISMISSED** [review]

**Review by IshikaM19 — APPROVED** [review]

**Pull request merged**